message SSL_DetectionBall {
  required uint32 area       = 1;
  required float x           = 2;
  required float y           = 3;
  required sint32 pixel_x    = 4;
  required sint32 pixel_y    = 5;
}

message SSL_DetectionRobot {
  required uint32 area       = 1;
  required float x           = 2;
  required float y           = 3;
  required sint32 pixel_x    = 4;
  required sint32 pixel_y    = 5;
}

message SSL_DetectionFrame {
  required uint32 frame_number = 1;
  required uint64 timestamp    = 2;
  required uint32 camera_id    = 3;
  repeated SSL_DetectionBall  balls  = 4;
  repeated SSL_DetectionRobot robots = 5;
}
